WorkplaceByFacebook/index/NormalConfig.json

```diff
@@ -216,7 +216,56 @@
               ]
             }
           ]
-        }
+        },
+        {
+          "Method": "GET",
+          "Path": "/community/events",
+          "QueryParameters": {
+          "fields": "id,cover,description,name,place,start_time"
+          },
+          "Headers": {
+          "Authorization": "Bearer @ApiKey"
+          },
+          "ItemPath": "data",
+          "ItemType": "WorkplaceEvent",
+          "Uri": "https://solutions788.workplace.com/events/%[id]/",

```

Comment:
solution788? Is that specific to a customer?

Reply:
Not really. It's just a test environment. I tried to keep the same for all configs to avoid multiple info.